app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/NavigationViewActivity.java

```diff
@@ -231,7 +229,7 @@ public void onResponse(Call<DirectionsResponse> call, Response<DirectionsRespons
         mapRoute.addRoutes(response.body().routes());
         boundCameraToRoute();
       } else {
-        Snackbar.make(mapView, "Please select a longer route", BaseTransientBottomBar.LENGTH_SHORT).show();
+        Snackbar.make(mapView, "Please select a longer route", Snackbar.LENGTH_SHORT).show();
```

Comment:
In order to increase readability, what about replacing magic numbers (literals in general including `String`s) with constants with a particular meaning?
Further information: [Replace Magic Number with Symbolic Constant](https://refactoring.com/catalog/replaceMagicNumberWithSymbolicConstant.html).